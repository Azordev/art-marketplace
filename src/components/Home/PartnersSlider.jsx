import React from "react";
import AyniPeruLogo from "../../assets/img/any-peru-logo.png";
import IWorkLogo from "../../assets/img/Logotipo-I-WORK.png";

const PartnersSlider = () => (
  <section className="bg-white px-5 lg:px-10 2xl:px-20 py-10 xl:py-16">
    <div className="flex flex-col space-y-8">
      <div className="flex flex-col items-center justify-center">
        <p className="text-xl sm:text-2xl xl:text-4xl font-bold">
          Nuestros colaboradores
        </p>
      </div>
      <div className="">
        <p className="text-xl font-normal text-center">
          Gracias a los convenios y alianzas estratégicas con las principales
          organizaciones de la industria de la construcción.
        </p>
      </div>
    </div>
    <div className="carousel relative bg-white">
      <div className="carousel-inner relative overflow-hidden w-full">
        <input
          className="carousel-open"
          type="radio"
          id="carousel-1"
          name="carousel"
          aria-hidden={true}
          hidden={true}
          defaultChecked="defaultChecked"
        />
        <div className="flex flex-col sm:flex-row carousel-item absolute opacity-0 w-full justify-evenly lg:justify-center lg:space-x-0 mb-10">
          <img
            src={AyniPeruLogo}
            alt="Ayni Peru Logo"
            className="block w-auto object-contain"
          />
          <img src={IWorkLogo} alt="IWork Logo" className="block w-auto object-contain" />
        </div>

        <input
          className="carousel-open"
          type="radio"
          id="carousel-2"
          name="carousel"
          aria-hidden={true}
          hidden={true}
          defaultChecked="defaultChecked"
        />
        <div className="flex flex-col sm:flex-row carousel-item absolute opacity-0 w-full justify-evenly lg:justify-center lg:space-x-0 mb-10">
          <img
            src={AyniPeruLogo}
            alt="Ayni Peru Logo"
            className="block w-auto object-contain"
          />
          <img src={IWorkLogo} alt="IWork Logo" className="block w-auto object-contain" />
        </div>

        <input
          className="carousel-open"
          type="radio"
          id="carousel-3"
          name="carousel"
          aria-hidden={true}
          hidden={true}
          defaultChecked="defaultChecked"
        />
        <div className="flex flex-col sm:flex-row carousel-item absolute opacity-0 w-full justify-evenly lg:justify-center lg:space-x-0 mb-10">
          <img
            src={AyniPeruLogo}
            alt="Ayni Peru Logo"
            className="block w-auto object-contain"
          />
          <img src={IWorkLogo} alt="IWork Logo" className="block w-auto object-contain" />
        </div>

        <ol className="carousel-indicators pt-40">
          <li className="inline-block mr-3">
            <label
              htmlFor="carousel-1"
              className="carousel-bullet cursor-pointer block text-4xl text-black hover:text-red-600"
            >
              •
            </label>
          </li>
          <li className="inline-block mr-3">
            <label
              htmlFor="carousel-2"
              className="carousel-bullet cursor-pointer block text-4xl text-black hover:text-red-600"
            >
              •
            </label>
          </li>
          <li className="inline-block mr-3">
            <label
              htmlFor="carousel-1"
              className="carousel-bullet cursor-pointer block text-4xl text-black hover:text-red-600"
            >
              •
            </label>
          </li>
        </ol>
      </div>
    </div>
  </section>
);

export default PartnersSlider;
