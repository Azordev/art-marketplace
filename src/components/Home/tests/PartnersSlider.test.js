import React from "react";
import renderer from "react-test-renderer";
import PartnersSlider from "../PartnersSlider";

it("renders correctly", () => {
  const tree = renderer.create(<PartnersSlider />).toJSON();
  expect(tree).toMatchSnapshot();
});
