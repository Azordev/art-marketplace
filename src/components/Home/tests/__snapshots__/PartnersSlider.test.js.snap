// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="bg-white px-5 lg:px-10 2xl:px-20 py-10 xl:py-16"
>
  <div
    className="flex flex-col space-y-8"
  >
    <div
      className="flex flex-col items-center justify-center"
    >
      <p
        className="text-xl sm:text-2xl xl:text-4xl font-bold"
      >
        Nuestros colaboradores
      </p>
    </div>
    <div
      className=""
    >
      <p
        className="text-xl font-normal text-center"
      >
        Gracias a los convenios y alianzas estratégicas con las principales organizaciones de la industria de la construcción.
      </p>
    </div>
  </div>
  <div
    className="carousel relative bg-white"
  >
    <div
      className="carousel-inner relative overflow-hidden w-full"
    >
      <input
        aria-hidden={true}
        className="carousel-open"
        defaultChecked="defaultChecked"
        hidden={true}
        id="carousel-1"
        name="carousel"
        type="radio"
      />
      <div
        className="flex flex-col sm:flex-row carousel-item absolute opacity-0 w-full justify-evenly lg:justify-center lg:space-x-0 mb-10"
      >
        <img
          alt="Ayni Peru Logo"
          className="block w-auto object-contain"
          src="any-peru-logo.png"
        />
        <img
          alt="IWork Logo"
          className="block w-auto object-contain"
          src="Logotipo-I-WORK.png"
        />
      </div>
      <input
        aria-hidden={true}
        className="carousel-open"
        defaultChecked="defaultChecked"
        hidden={true}
        id="carousel-2"
        name="carousel"
        type="radio"
      />
      <div
        className="flex flex-col sm:flex-row carousel-item absolute opacity-0 w-full justify-evenly lg:justify-center lg:space-x-0 mb-10"
      >
        <img
          alt="Ayni Peru Logo"
          className="block w-auto object-contain"
          src="any-peru-logo.png"
        />
        <img
          alt="IWork Logo"
          className="block w-auto object-contain"
          src="Logotipo-I-WORK.png"
        />
      </div>
      <input
        aria-hidden={true}
        className="carousel-open"
        defaultChecked="defaultChecked"
        hidden={true}
        id="carousel-3"
        name="carousel"
        type="radio"
      />
      <div
        className="flex flex-col sm:flex-row carousel-item absolute opacity-0 w-full justify-evenly lg:justify-center lg:space-x-0 mb-10"
      >
        <img
          alt="Ayni Peru Logo"
          className="block w-auto object-contain"
          src="any-peru-logo.png"
        />
        <img
          alt="IWork Logo"
          className="block w-auto object-contain"
          src="Logotipo-I-WORK.png"
        />
      </div>
      <ol
        className="carousel-indicators pt-40"
      >
        <li
          className="inline-block mr-3"
        >
          <label
            className="carousel-bullet cursor-pointer block text-4xl text-black hover:text-red-600"
            htmlFor="carousel-1"
          >
            •
          </label>
        </li>
        <li
          className="inline-block mr-3"
        >
          <label
            className="carousel-bullet cursor-pointer block text-4xl text-black hover:text-red-600"
            htmlFor="carousel-2"
          >
            •
          </label>
        </li>
        <li
          className="inline-block mr-3"
        >
          <label
            className="carousel-bullet cursor-pointer block text-4xl text-black hover:text-red-600"
            htmlFor="carousel-1"
          >
            •
          </label>
        </li>
      </ol>
    </div>
  </div>
</section>
`;
